docs/v3_services.rst

```diff
@@ -347,3 +376,14 @@ When you're ready, follow the steps below to transition to v3 services only:
 
     ./securedrop-admin sdconfig   # choose "no" for v2, "yes" for v3
     ./securedrop-admin tailsconfig
+
+
+Human-readable onion URLs
```

Comment:
Thanks for adding this section! The feature name (as also referenced in the Tor browser UI if you click on extended information for an onion address) is "onion name", which I think we should use at least once in this paragraph.

Since it's likely to come up again, what do you think about maybe explicitly adding a sentence here about "vanity addresses", e.g.:

> While it is possible to generate "vanity addresses" that start with a few recognizable letters, we are unable to provide technical assistance with any such process, and we recommend registering an onion name instead.

Reply:
Added 'onion names' terminology in 61fa02a, and also added

> Onion names are the supported solution for organizations wishing to 
> customize their *Source Interface* URL, and are recommended over "vanity"
> URLs (addresses that start with a few recognizable characters).